src/stories/Filter/components/ContextSwitchMenu.tsx

```diff
@@ -0,0 +1,80 @@
+import { FC, useState } from 'react';
+import styled from '@emotion/styled';
+import { ThemeProvider } from '@emotion/react';
+import theme from 'src/styles/theme';
+import SelectPopover, { SelectPopoverOption } from 'src/SelectPopover/SelectPopover';
+import Button from 'src/Button/Button';
+import { ContextSwitchMenuProps } from 'src/Filter/types';
+import { ReactComponent as DownArrow } from './assets/downArrow.svg';
+
+const ContextSwitchContainer = styled.div({
+    alignSelf: 'center'
+});
+
+const DropdownButton = styled(Button)(({ theme }) => ({
+    width: 180,
+    height: 36,
+    paddingLeft: 13,
+    padding: 0,
+    border: `0px solid white`,
+    fontWeight: 400,
+    justifyContent: 'space-between',
+    ':hover': {
+        backgroundColor: theme.colors.white
+    },
+    ':after': {
+        transition: 'transform 0s, opacity 0s'
+    }
+}));
+
+const ButtonLabel = styled.div({
```

Comment:
I think you could move these styles to a separate contextSwitchMenuStyles.ts file like we do for other components in lakefront